torch/testing/_internal/dist_utils.py

```diff
@@ -26,12 +26,19 @@ def __init__(self, *args, **kwargs):
 INIT_METHOD_TEMPLATE = "file://{file_name}"
 
 
-def dist_init(old_test_method=None, setup_rpc=True, clean_shutdown=True):
+def dist_init(old_test_method=None, setup_rpc=True, clean_shutdown=True,
+              faulty_messages=None):
     """
     We use this decorator for setting up and tearing down state since
     MultiProcessTestCase runs each `test*` method in a separate process and
     each process just runs the `test*` method without actually calling
     'setUp' and 'tearDown' methods of unittest.
+
+    Note: pass the string representation of MessageTypes that should be used
+    with the faulty agent's send function. By default, all retriable messages
```

Comment:
isn't the default value for `faulty_messages` `None`?

Reply:
I see, it's set in `test_faulty_agent_rpc_trap.py`. How does this trap work for OSS? And does it mean all our normal tests also sees the same `faulty_messages` in their `dist_util`?